angular/CHANGELOG.md

```diff
@@ -57,12 +57,13 @@
     produced accidentally created two queries instead of one.
     [Closes #1455][#1455].
```

Comment:
@alorenzen: in general when referring to an issue number, how about **simply writing the issue number in parentheses** at the end of the bullet rather than `Closes #1234`? It would be more concise and amount to less typing.